pimd/pim_instance.h

```diff
@@ -165,6 +165,7 @@ struct pim_instance {
 	struct route_table *rp_table;
 
 	int iface_vif_index[MAXVIFS];
+	int mcast_if_count;
```

Comment:
It is probably a good idea to display this count via some command (for future debugging purposes). We don't seem to have "pim summary" command to show mroute# etc. Maybe time to add such a command?

Reply:
pretty sure this is there w/ show ip multicast